modules/appeals_api/spec/services/appeals_api/central_mail_updater_spec.rb

```diff
@@ -106,21 +106,14 @@
 
     describe 'correctly maps packet statuses' do
       describe 'with one returned packet' do
-        it 'completedReason == UnidentifiableMail' do
+        it 'status == Error' do
           central_mail_response = [
             {
               uuid: appeal_1.id,
-              status: 'Complete',
-              errorMessage: '',
+              status: 'Error',
+              errorMessage: 'Errors: ERR-EMMS-FAILED, Log Date cannot be in the future.',
               lastUpdated: '2018-04-25 00:02:39',
-              packets: [
-                {
-                  veteranId: appeal_1.ssn,
-                  status: 'Complete',
-                  completedReason: 'UnidentifiableMail',
-                  transactionDate: Time.zone.today
-                }
-              ]
+              packets: []
```

Comment:
We didn't have any unit tests specifically for `status: 'Error'`, and since the test that I added below checks both the `status` and `detail` for "UnidentifiableMail" submissions, I thought it made sense to repurpose this one.